archetypes/widgets/experiences.md

```diff
@@ -0,0 +1,44 @@
+---
+title: {{ replace .Name "-" " " | title }}
+draft: false
+experiences:
+  - title:
+    organization:
+      name:
+      url:
+    dates:
+    location:
+    writeup:
+
+  - title:
+    organization:
+      name: 
+      url:
+    dates:
+    location:
+    writeup:
```

Comment:
I think we can just leave one item here and people can add themselves.

Reply:
Sounds good! Commit added